pkg/kubectl/cmd/util/printing.go

```diff
@@ -53,7 +53,7 @@ func AddNonDeprecatedPrinterFlags(cmd *cobra.Command) {
 	cmd.MarkFlagFilename("template")
 	cmd.Flags().String("sort-by", "", "If non-empty, sort list types using this field specification.  The field specification is expressed as a JSONPath expression (e.g. '{.metadata.name}'). The field in the API resource specified by this JSONPath expression must be an integer or a string.")
 	cmd.Flags().BoolP("show-all", "a", true, "When printing, show all resources (default show all pods including terminated one.)")
-	cmd.Flags().MarkDeprecated("show-all", "will be removed in an upcoming release")
+	cmd.Flags().MarkDeprecated("show-all", "and it is inert now. It will be removed in an upcoming release")
```

Comment:
We're just going to remove it in this release, so there's no need to update this description further.

Reply:
I think we should remove these description:
> When printing, show all resources (default show all pods including terminated one.

Maybe:
```
cmd.Flags().BoolP("show-all", "a", true, "DEPRECATED")
```

So that when users run `kubectl get -h`, users will get the newest information.